### PR TITLE
Zoombutton plugin

### PR DESCRIPTION
**Pull request by moovida**
This adds a possible answer to issue #481. 

* commit 1: adds a few files that came while enabling the web part in the flutter example project. The issue mentioned web, so web has been tested
* commit 2: adds a simple zoombuttons plugin that shows 2 FAB buttons that act as zoom-in/out buttons. And an entry in the drawer (**ZoomButtons Plugin**) to view them in action.

### CODE REVIEW

File: example/ios/Runner/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+@UIApplicationMain
+@objc class AppDelegate: FlutterAppDelegate {
+  override func application(
+    _ application: UIApplication,
+    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+  ) -> Bool {
+    GeneratedPluginRegistrant.register(with: self)
+    return super.application(application, didFinishLaunchingWithOptions: launchOptions)
+  }
+}
```
Comment (johnpryan) [inline example/ios/Runner/AppDelegate.swift]: extra files?

File: example/web/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>example</title>
+</head>
+<body>
+  <script src="main.dart.js" type="application/javascript"></script>
+</body>
+</html>
```
Comment (johnpryan) [inline example/web/index.html]: extra file?

File: example/.gitignore [hunk 3]
```diff
@@ -9,3 +9,4 @@ ios/.symlinks/
 packages
 pubspec.lock
 .flutter-plugins
+*.iml
```
Comment (johnpryan) [inline example/.gitignore]: use a [global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for your flavor of IDE instead

File: example/ios/Runner/AppDelegate.swift (hunk 1 above)
Comment (moovida) [reply example/ios/Runner/AppDelegate.swift]: It was generated when I added web support. But I guess that one can be removed. Will do

File: example/web/index.html (hunk 2 above)
Comment (moovida) [reply example/web/index.html]: This is actually the base for the web project, without which you can't run in web mode. So I will leave it.

File: example/.gitignore (hunk 3 above)
Comment (moovida) [reply example/.gitignore]: Ha, I learned something new and very cool. Thanks. Will do.

**Comment by moovida** [comment]
Hope it is ok to do a revert of the web files commit and add back only the index.html.

**Comment by moovida** [comment]
@johnpryan , I am not sure why this is not accepted. I fixed all the requests you made apart of the file necessary for web. Is there something you are expecting me to do?

**Comment by johnpryan** [comment]
@moovida I think I was waiting for the CI issues to be fixed. I went ahead and fixed them in ba9189203ce211f6cb3ef84355fd90502bbe2dc6.

**Comment by moovida** [comment]
@johnpryan , I apologize. I am not sure I understand what I did wrong (I saw the changes you had to do). Did I format by error? Just let me know so that it doesn't happen in future.

**Comment by johnpryan** [comment]
I think it was just an issue with running `dartanalyzer .` and `dartfmt -w .`

**Comment by yvvivek9** [comment]
very useful plugin but how to use it, nothing is mentioned on documentation website or pub.dev

**Comment by josxha** [comment]
Woah that's a comment on an old thread... 
@yvvivek9 Take a look at the changes, the plugin has been added to the example app and not the flutter_map package itself. Plugins are in general not part of the flutter_map package.

You can check it out in the example app https://demo.fleaflet.dev/plugin_zoombuttons or the source code at https://github.com/fleaflet/flutter_map/blob/master/example/lib/plugins/zoombuttons_plugin.dart

**Pull request merged**